Simplified/NYPLErrorLogger.swift

```diff
@@ -450,7 +450,7 @@ fileprivate let nullString = "null"
     let err = error ?? NSError.init(domain: simplyeDomain,
                                     code: ErrorCode.userProfileDocFail.rawValue,
                                     userInfo: nil)
-    var metadata = [AnyHashable : Any]()
+    var metadata: [AnyHashable : Any] = error?.userInfo ?? [AnyHashable : Any]()
```

Comment:
the error.userInfo is already reported by the subsequent call to `recordError` and its values merged with the additional metadata hash, so I think this isn't necessary.

Reply:
fixed